core/src/preferences/PreferencesBackend.cpp

```diff
@@ -194,17 +194,47 @@ namespace ledger {
             unsetEncryption();
 
             auto emptySalt = std::string("");
-            auto pref = getPreferences("__core");
-            auto salt = pref->getString(ENCRYPTION_SALT_KEY, emptySalt);
+            auto saltKey = std::vector<uint8_t>(ENCRYPTION_SALT_KEY.cbegin(), ENCRYPTION_SALT_KEY.cend());
```

Comment:
Make saltKey static.

Reply:
Why? 🙂 You want to keep around memory that is likely to be used only once in a whole application lifetime?

Reply:
We have this code in several places, so I want to be sure we always use the same key. This is just few bytes...